src/actions/ui.js

```diff
@@ -24,6 +24,10 @@ export function closeActiveSearch() {
   };
 }
 
+export function closeProjectSearch() {
```

Comment:
No need for the duplicate. Just the one above is fine. The `type` is still the same.